tracking/tracking.go

```diff
@@ -0,0 +1,40 @@
+package tracking
+
+import (
+	"os"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/google/uuid"
+	rancher "github.com/rancher/go-rancher/v2"
+)
+
+const UUIDSetting = "install.uuid"
+
+var RancherUUID = loadRancherUUID()
```

Comment:
I think it would be better to make `loadRancherUUID` a public function and then call it from `main()` to get the value. Then you can make a new member variable for `manager.Manager` and set that in `main()`.